normandy/selfrepair/tests/test_normandy_driver.js

```diff
@@ -0,0 +1,38 @@
+import NormandyDriver from '../static/js/normandy_driver.js';
+
+describe('Normandy Driver', () => {
+  describe('showHeartbeat', () => {
+    it('should pass all the required arguments to the UITour helper', async function() {
```

Comment:
Lint: prefer arrow functions. Eslint has some issues with `async () => {}`, but it is valid. If this is problematic for you, leave it and I'll fix it in my linter branch (which can deal with `async () => {}`).